ShapesProject/src/main/java/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+public class Rectangle extends Shape {
+    private final double width, length; //sides
+
+    public Rectangle() {
```

Comment:
why? this is unexpected behavior. 2,3 are just random numbers. you should not allow a default constructor